samples/analyze.v1p3beta1.js

```diff
@@ -0,0 +1,380 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function detectPerson(path) {
+  //[START video_detect_person_beta]
+  // Imports the Google Cloud Video Intelligence library + Node's fs library
+  const Video = require('@google-cloud/video-intelligence').v1p3beta1;
+  const fs = require('fs');
+  const util = require('util');
+  // Creates a client
+  const video = new Video.VideoIntelligenceServiceClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const path = 'Local file to analyze, e.g. ./my-file.mp4';
+
+  // Reads a local video file and converts it to base64
+  const file = await util.promisify(fs.readFile)(path);
+  const inputContent = file.toString('base64');
+
+  const request = {
+    inputContent: inputContent,
+    features: ['PERSON_DETECTION'],
+    videoContext: {
+      personDetectionConfig: {
+        // Must set includeBoundingBoxes to true to get poses and attributes.
+        includeBoundingBoxes: true,
+        includePoseLandmarks: true,
+        includeAttributes: true,
+      },
+    },
+  };
+  // Detects people in a video
+  const [operation] = await video.annotateVideo(request);
+  const results = await operation.promise();
+  console.log('Waiting for operation to complete...');
+
+  // Gets annotations for video
+  const personAnnotations =
+    results[0].annotationResults[0].personDetectionAnnotations;
+
+  personAnnotations.forEach(personAnnotation => {
```

Comment:
to avoid the `forEach`, and an extra layer of callbacks, you could do this: 

```js
for (const {segment, timestampedObjects} of tracks) {
  console.info(segment, timestampedObjects);
}
```

I've been tending to use this syntax rather than `forEach` these days, it's also nice because you can break out of the loop with `break`.

Reply:
Done, here and elsewhere. 

Thank you for showing me this syntax!